styles/fonts.css

```diff
@@ -0,0 +1,45 @@
+/* montserrat-300 - latin */
+@font-face {
+  font-family: "Montserrat";
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: url("/fonts/montserrat-v15-latin-300.eot");
+  src: local(""),
+    url("/fonts/montserrat-v15-latin-300.eot?#iefix")
+      format("embedded-opentype"),
+    url("/fonts/montserrat-v15-latin-300.woff2") format("woff2"),
+    url("/fonts/montserrat-v15-latin-300.woff") format("woff"),
+    url("/fonts/montserrat-v15-latin-300.ttf") format("truetype"),
+    url("/fonts/montserrat-v15-latin-300.svg#Montserrat") format("svg");
+}
+/* montserrat-500 - latin */
+@font-face {
+  font-family: "Montserrat";
+  font-style: normal;
+  font-weight: 500;
+  font-display: swap;
+  src: url("/fonts/montserrat-v15-latin-500.eot");
+  src: local(""),
+    url("/fonts/montserrat-v15-latin-500.eot?#iefix")
+      format("embedded-opentype"),
+    url("/fonts/montserrat-v15-latin-500.woff2") format("woff2"),
+    url("/fonts/montserrat-v15-latin-500.woff") format("woff"),
+    url("/fonts/montserrat-v15-latin-500.ttf") format("truetype"),
+    url("/fonts/montserrat-v15-latin-500.svg#Montserrat") format("svg");
+}
+/* montserrat-600 - latin */
```

Comment:
Please add a black space between the `@font-face` declarations to be consistent with the rest of the codebase